app/serializers/reconciliation_csv_serializer.rb

```diff
@@ -5,14 +5,14 @@ class ReconciliationCSVSerializer < CausalityCurvesCSVSerializer
   private
 
   def producer_types
-    %i[producer import storage]
+    %i[producer flex]
   end
 
   def consumer_types
-    %i[consumer export storage]
+    %i[consumer flex]
```

Comment:
Why are export and storage removed here? These attributes still exist on ETSource for the natural gas nodes. For example `energy_export_natural_gas_backup` has the following:
```
- network_gas.profile = network_gas_export
```

Reply:
@kndehaan can you maybe check this question, as you were the one working on this?

Reply:
I thought that this class was only used for the hydrogen csv, but now see that it's also used for the network gas csv, my bad. Changes reverted and class description updated with https://github.com/quintel/etengine/pull/1455/commits/be650f4e7da43326860f92cad370b430759c90bf.